python-connectors/dss-plugin-stripe-object/connector.py

```diff
@@ -0,0 +1,101 @@
+
+import json
+from dataiku.connector import Connector
+import requests
+from requests.auth import HTTPBasicAuth
+
+
+def log(msg):
+    print('Stripe plugin - %s' % msg)
+
+
+def iterate_dict(dictionary, parents=[]):
+    """
+    This function iterates over one dict and returns a list of tuples: (list_of_keys, value)
+    Usefull for looping through a multidimensional dictionary.
+    """
+
+    ret = []
+    for key, value in dictionary.iteritems():
```

Comment:
iteritems does not exists in python 3. Use items instead.

Reply:
Well spotted.